src/passes/userDefinedTypesConverter.ts

```diff
@@ -57,8 +59,21 @@ export class UserDefinedTypesConverter extends ASTMapper {
           node.raw,
         ),
       );
+    } else if (typeNode instanceof FunctionType) {
+      const newTypeString: string = generateExpressionTypeString(typeNode);
+
+      ast.replaceNode(
+        node,
+        new Identifier(
+          node.id,
+          node.src,
+          newTypeString,
```

Comment:
Better to just assign to the typestring of the existing node here (and in the case above) rather than replacing it with a node of the same type with most properties the same

Reply:
Implemented 👍